server.js

```diff
@@ -3636,6 +3674,11 @@ function fetchFromSvg(request, url, cb) {
   });
 }
 
+function ordinalNumber(n) {
+  var s=["ᵗʰ","ˢᵗ","ⁿᵈ","ʳᵈ"], v=n%100;
```

Comment:
Yes... Do you have another suggestion? Normal size letters don't look good
either...

On Sun, 3 May 2015 16:03 Thaddee Tyl <notifications@github.com> wrote:

> In server.js
> <https://github.com/badges/shields/pull/433#discussion_r29556370>:
>
> > @@ -3636,6 +3674,11 @@ function fetchFromSvg(request, url, cb) {
> >    });
> >  }
> >
> > +function ordinalNumber(n) {
> > +  var s=["ᵗʰ","ˢᵗ","ⁿᵈ","ʳᵈ"], v=n%100;
>
> That's pretty cool, but I fear it won't render well:
> [image: nth]
> <https://cloud.githubusercontent.com/assets/100689/7445323/ee809eda-f1ad-11e4-9609-a430a408892a.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/badges/shields/pull/433/files#r29556370>.
>